### ISSUE TITLE
GraphQL error after installing Formie plugin

### ISSUE THREAD

**Issue by andreasottosson** [craftcms/gatsby-source-craft#52]
### Description

Installing Formie throws the following error. We have tried enabling all "Formie" related checkboxes under GQL schema settings but we won't use that, anyway enabling them just throws another error. If we disable Formie the error goes away.

```
success fetching Form - 12.772s

 ERROR #11330  PLUGIN

"gatsby-source-craft" threw an error while running the sourceNodes lifecycle:

Variable "$id" is not defined

  468 |     if (remoteConfigVersion !== localConfigVersion || !localContentUpdateTime) {
  469 |         reporter.info("Cached content is unavailable or outdated, sourcing _all_ nodes.");
> 470 |         await sourceAllNodes(config);
      |         ^
  471 |     }
  472 |     else {
  473 |         reporter.info(`Craft config version has not changed since last sourcing. Checking for content changes since "${localContentUpdateTime}".`);

Make sure that you don't have a typo somewhere and use valid arguments in sourceNodes lifecycle.
Learn more about sourceNodes here: https://www.gatsbyjs.com/docs/reference/config-files/gatsby-node/#sourceNodes

File: node_modules/gatsby-source-craft/gatsby-node.js:470:9

success Checking for changed pages - 0.001s
```

This is the error with all checkboxes enabled under "Formie" in the GraphQL settings.

```
 ERROR #11321  PLUGIN

"gatsby-node.js" threw an error while running the createPages lifecycle:

Interface field Craft_FormInterface.fields expected but Craft_Form does not provide it.

  4 |   const { createPage } = actions
  5 |
> 6 |   const result = await graphql(`
    |                        ^
  7 |     query {
  8 |       allCraftPagesStandardEntry {
  9 |         nodes {

File: gatsby-node.js:6:24



  Error: Interface field Craft_FormInterface.fields expected but Craft_Form does not provide it.
```

### Steps to reproduce

1. Install Formie
2. Run "gatsby develop"

### Additional info

- Craft version: 3.7.16
- PHP version: 7.3.31
- Database driver & version: MySQL 5.6.51
- Plugins & versions: 
Cloner | 1.2.2
Formie | 1.4.20
Gatsby Helper | 1.0.5
Redactor | 2.8.8
Smith | 1.1.14
Spoon | 3.6.1
Sprout Notes | 2.2.3
Tags | 1.0.9

**Comment by andris-sevcenko** [comment]
Related: https://github.com/verbb/formie/issues/343

**Comment by andris-sevcenko** [comment]
This should be resolved after u[dating Formie to 1.4.21
